actions/info.action.ts

```diff
@@ -39,7 +39,7 @@ export class InfoAction extends AbstractAction {
 
   private async displaySystemInformation(): Promise<void> {
     console.info(chalk.green('[System Information]'));
-    console.info('OS Version     :', chalk.blue(osName(platform(), release())));
+    console.info('OS Version     :', chalk.blue(osName(platform(), release()), release()));
```

Comment:
```suggestion
    console.info('OS Version     :', chalk.blue(osName(platform(), release())));
```

Reply:
@Tony133 this will log `OS Version : MacOs` instead of `OS Version : MacOs 23.0.0`

Reply:
Ah okay, it looks like a repetition of the method